libs/model/src/models/membership.ts

```diff
@@ -36,9 +36,13 @@ export default (
   const Membership = <MembershipModelStatic>sequelize.define(
     'Membership',
     {
-      id: { type: dataTypes.INTEGER, primaryKey: true, autoIncrement: true },
-      group_id: { type: dataTypes.INTEGER, allowNull: false },
-      address_id: { type: dataTypes.INTEGER, allowNull: false },
+      id: { type: dataTypes.INTEGER, allowNull: false },
```

Comment:
get this error if enable autoIncrement:

'ALTER TABLE "Memberships" ALTER COLUMN "id" DROP NOT NULL;ALTER TABLE "Memberships" ALTER COLUMN "id" DROP DEFAULT;ALTER TABLE "Memberships" ALTER COLUMN "id" TYPE   SERIAL;',

type "serial" does not exist

Maybe we don't need this detail in the static model? @timolegros?

Reply:
@Rotorsoft It doesn't seem that the migrations create a primary key on the memberships table. Additionally, the migrations create foreign keys on `group_id` and `address_id` which doesn't seem to be the case with `.sync`. I do think we need autoIncrement otherwise we will need to manually input an id in tests...

Reply:
OK. Reverted the changes in `membership.ts` and just added the `address_id` index that is part of the migrations... also added the `id` PK to the exceptions (this is the sequelize known issue with mandatory id PK)... we should revisit this model in a second round. @timolegros